spec/controllers/media_controller_spec.rb

```diff
@@ -366,6 +379,17 @@ def download
         end
       end
 
+      context "when a user is not authenticated and has provided a valid token with draft_asset_manager_access" do
+        before { not_logged_in }
+
+        it "grants access to the file" do
+          expect(controller).not_to receive(:authenticate_user!)
+          query_params = params.tap { |p| p[:params].merge!(token: token_with_draft_asset_manager_access) }
```

Comment:
Couldn't this be simplified to this...?

```ruby
          query_params = params.merge(token: token_with_draft_asset_manager_access)
```

Reply:
I'm getting `unknown keyword: token` 
I copied that line from another test 🙈

Reply:
Ah I misunderstood what it's doing. It's merging `token` into the `:params` field of the `params` hash - but my code sample would merge `:token` as a top level field. The use of `tap` makes sense now.